### PR TITLE
Removes a bunch of in world

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Replaces with in machines.

Seemed to still function when I did breaking lights and giving all doors maint access.

If this isn't the standard than I apologize, I was told to use in machines instead of in world in another PR, ran into a bunch of them and figured I'd clean it up.

### CODE REVIEW

File: code/modules/projectiles/projectile/beams.dm [hunk 1]
```diff
@@ -93,3 +93,17 @@
 /obj/item/projectile/lasertag/bluetag
 	icon_state = "bluelaser"
 	suit_types = list(/obj/item/clothing/suit/redtag)
+
+
+/obj/item/projectile/beam/accelerator
```
Comment (Fox-McCloud) [inline code/modules/projectiles/projectile/beams.dm]: what does this have to do with the PR?

Comment (KorPhaeron) [inline code/modules/projectiles/projectile/beams.dm]: My inability to untick files mostly

**Comment by Fox-McCloud** [comment]
all APCs can be replaced with "in apcs" as far as I know---as that list is smaller than even machines.

**Comment by optimumtact** [comment]
Who wrote the original code :hankey:

File: code/game/gamemodes/events.dm [hunk 2]
```diff
@@ -103,7 +103,7 @@
 		A.power_environ = 0
 		A.power_change()
 
-	for(var/obj/machinery/power/apc/C in world)
+	for(var/obj/machinery/power/apc/C in machines)
```
Comment (RemieRichards) [inline code/game/gamemodes/events.dm]: ```apcs_list```

File: code/modules/admin/verbs/debug.dm [hunk 3]
```diff
@@ -546,37 +546,37 @@ var/global/list/g_fancy_list_of_types = null
 		if(!(A.type in areas_all))
 			areas_all.Add(A.type)
 
-	for(var/obj/machinery/power/apc/APC in world)
+	for(var/obj/machinery/power/apc/APC in machines)
```
Comment (RemieRichards) [inline code/modules/admin/verbs/debug.dm]: do ```in apcs_list```

File: code/modules/research/message_server.dm [hunk 4]
```diff
@@ -276,7 +276,7 @@ var/obj/machinery/blackbox_recorder/blackbox
 	var/pda_msg_amt = 0
 	var/rc_msg_amt = 0
 
-	for (var/obj/machinery/message_server/MS in world)
+	for (var/obj/machinery/message_server/MS in machines)
```
Comment (Giacom) [inline code/modules/research/message_server.dm]: You can change this to 'message_servers'.

**Comment by KorPhaeron** [comment]
Should be ready now @RemieRichards
